errbot/backends/slack.py

```diff
@@ -394,17 +417,21 @@ def _message_event_handler(self, event):
 
     def userid_to_username(self, id_):
         """Convert a Slack user ID to their user name"""
-        user = [user for user in self.sc.server.users if user.id == id_]
+        users = self.api_call('users.list')
```

Comment:
You changed the contract of the method doing that and maybe broke stuff down the line:
When I compare the 2 lists:
![image](https://cloud.githubusercontent.com/assets/975564/16163361/a4c39cb0-348d-11e6-8237-de11bf2d0a28.png)
and
![image](https://cloud.githubusercontent.com/assets/975564/16163401/d5219150-348d-11e6-91a2-20d854a484ca.png)